app/main/controllers/activity/RTMediaActivity.php

```diff
@@ -34,16 +34,21 @@ function __construct( $media, $privacy = 0, $activity_text = false ) {
 		$this->privacy       = $privacy;
 	}
 
+	/**
+	 * Create html of activity/comments using text and media.
+	 * 
+	 * @param string $type Type of activity (activity/comment).
+	 */
```

Comment:
Please write about return type, and can we add more description about this function.